extensions/wikia/UserLogin/tests/UserLoginTest.php

```diff
@@ -441,6 +443,8 @@ public function testChangePassword( $params, $mockWebRequestParams, $mockWgUserP
 	}
 
 	public function changePasswordDataProvider() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function "changePasswordDataProvider" has 167 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)